uw-frame-components/staticFeeds/sample-messages.json

```diff
@@ -86,6 +91,7 @@
       "expireDate": "2017-08-15",
       "featureImageUrl": "img/the-cow.png",
       "priority": "high",
+      "notDismissible": false,
```

Comment:
Let's not with the negative in the name of the property. Maybe `"userDismissible": true,` ?

Reply:
My thinking here was that the default case should be that dismissible is true, since the vast majority of notifications are not important enough to linger. The attribute should be optional and should only be set when something's important enough to be non-dismissible -- otherwise folks shouldn't have to bother with it. I agree that the negative property is a bad look. What if we used something to reflect non-dismissibility, like "permanent" or "sticky"?